src/shared/styles/_fonts.scss

```diff
@@ -0,0 +1 @@
+$roboto-font-family: "Roboto";
```

Comment:
I would suggest using something like `$primary-font-family` because the idea of having a global variable is that it is configurable, so in the future, if the primary font family changes you can do it easily, but if the name is related to the value it won't make sense have `$roboto-font-family: "Poppins";` for example.